extensions/amp-sidebar/1.0/toolbar.js

```diff
@@ -47,6 +50,9 @@ export class Toolbar {
     /** @private {Element|undefined} */
     this.targetElement_ = undefined;
 
+    /** @private {Element|undefined} */
+    this.placeholder_ = undefined;
```

Comment:
I added the placeholder, but it does not remove the need for keeping a reference to the placeholder. Since we are resizing it's height as needed.

However, with accordance to your other comment about adding and removing the placeholder dynamically on attemptShow(), this would make sense to move since we would be creating it there, and not in the toolbar build callback. I'll remove this if we decide to go that route instead of hiding the parent container we create.